_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Micro-procurement for digital work - what's stopping us?"
+ref: working-in-the-open-p1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC and TBS"
+date: "2020-07-28"
+excerpt_separator: <!--more-->
+
+---
+
+The Government of Canada (GC) buys a lot of stuff. In 2016, the total spent by the GC on goods and services was ~$15.9B. From an IT point
+of view, this spending can include big ticket items like new data centres and new national applications costing tens of millions of
+dollars, but can also include low dollar value items like paying someone to build a small prototype or write a position paper.
```

Comment:
Propose a solution here? If this is the summary, we've stated the context, but not why they should read the blog

Reply:
Hmmm. I wish there was a way to write a summary excerpt without changing the flow of the blog. Like in the meta data.

Reply:
You can totally put something in the Front matter by doing the following:
1. Remove the `<!--more-->` separator from the body of your post
2. Insert a new `excerpt:` variable in the front matter
3. Fill in the content of the variable with the content of your choice in double quote marks `"`. 

exemple:

`excerpt: "This is my new excerpt, which is not part of the main article"`

Reply:
Fixed... I added a blurb at the top about this being part one of a two part series and that functioned well as the excerpt. Let me know what you think.